wasmcloud-test-util/src/lib.rs

```diff
@@ -8,66 +8,8 @@ pub mod cli;
 pub use nkeys;
 pub use regex;
 
-pub mod testing {
-
-    smithy_bindgen::smithy_bindgen!("testing/testing.smithy", "org.wasmcloud.interface.testing");
-
-    // after sdk is split we won't have to duplicate this code
-    impl Default for TestOptions {
```

Comment:
I think losing these will break downstream consumers, e.g. https://github.com/wasmCloud/capability-providers/blob/0144fea03d2eebfe8b3c91bd381b400a8ce2479a/nats/tests/nats_test.rs/#L14

Reply:
I'll roll this into the downstream crates check

Reply:
Yup this does break the downstream consumers -- They all have to be upgraded together.

Getting the NATS provider to build required the following changes:

in `wasmcloud-provider-nats`:
- `async-nats=0.30`
- `wasmbus-rpc=0.14`
- `wasmcloud-interface-messaging=custom`
- `wasmcloud-test-util=custom`

in `wasmcloud-interface-messaging=custom`:
- `wasmbus-rpc=0.14`

in `wasmcloud-test-util`:
- `wasmcloud-interface-testing=custom`
- `wasmbus-rpc=0.14`

in `wasmcloud-interface-testing`:
- `wasmbus-rpc=0.14`

IIRC there was going to be push to bump all of these.

Long term I think we need to remove some of the macros and start using some traits mroe,